content/using/install.md

```diff
@@ -164,39 +139,37 @@ The `>=` output means that a command was successful. Now you can see your ship's
 
 ### Using Landscape
 
-Landscape is the Urbit web interface, and it's the best way to interact with your ship. Chrome, Firefox, and Brave are the recommended browsers for using Landscape. To get onto Landscape:
+Landscape is the Urbit web interface, and it's the best way to interact with your ship. To access Landscape:
 
 1. Start your ship. In the boot messages, look for a line that says something like `http: live (insecure, public) on 80`. The number given is the port that your ship is using.
 2.  If the port given is 80, simply type `localhost` into your browser's address bar. If the given port is a different number, such as `8080`, you would type `localhost:8080`. You'll be met with a login prompt.
 3. Type `+code` into your comet's Dojo. Copy-paste the returned code into the field asking for it in the browser page.
-4. You're in! Now you can explore apps such as Chat for messages, Publish for blogging, and Weather.
+4. You're in! Now you can explore apps such as Chat, Publish, and more.
 
-### Join a Chat
+### Join a group
 
-Now that you're on landscape, let's join the Urbit Community chat, a great place for newcomers to ask questions.
+Now that you're on Landscape, join the Urbit Community, a great place for newcomers to ask questions.
 
-Click that `Chat` type on your Landscape home page, and then click the `Join Chat` column. Then, enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Chat` button.
+Click the 'Groups' button on your Landscape home page, and then click the `Join Group` column. Then, enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Group` button.
 
-You can send and receive messages here, just like any other chatroom.
+From the Urbit Community group you can join a variety of chatrooms, notebooks, and collections.
 
-## Getting an Urbit ID (Planet or Star)
+## Setting up a planet {#id}
+
+If you have an Urbit ID, you'll use the [Bridge](https://bridge.urbit.org) client to get your ship's *keyfile* before you can [boot your ship](#booting-your-ship).  Your keyfile is a cryptographic secret that allows your Urbit instance to authenticate itself on the network, so you can't boot your planet without it. Your keyfile is deterministically derived from your other cryptographic secrets.
```

Comment:
I think the way this is written might give some people the idea that you need to use your keyfile for subsequent boots (I know we clarify this later but I generally don't want to assume people read the whole thing)

```suggestion
If you have an Urbit ID, you'll use the [Bridge](https://bridge.urbit.org) client to get your ship's *keyfile* before you can [boot your ship](#booting-your-ship).  Your keyfile is a cryptographic secret that allows your Urbit instance to authenticate itself on the network, so it is required the first time you boot your ship. Your keyfile is deterministically derived from your other cryptographic secrets.
```